src/Error/UserError.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace GraphQL\Error;
+
+/**
+ * Class UserError
+ *
+ * Note:
+ * Error that can be display safely to client...
+ *
+ * @package GraphQL\Error
+ */
+class UserError extends InvariantViolation
```

Comment:
BC is never acceptable or not it is just a reality, if your new release reflect the BC no trouble but it is very important to dependencies...